sdk/digitaltwins/digitaltwins/samples/javascript/dt_component_lifecycle.js

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+const { DefaultAzureCredential } = require("@azure/identity");
+const { DigitalTwinsClient } = require("@azure/digitaltwins");
+const { v4 } = require("uuid");
+
+// Scenario example of how to:
+// - create a DigitalTwins Service Client using the DigitalTwinsClient constructor
+// - create model, component and twin
+// - create digital twin based on the model
+// - update component
+// - get component
+// - delete twin
+// - decomission and delete model, component
+//
+// Preconditions:
+// - Environment variables have to be set
+// - DigitalTwins enabled device must exist on the ADT hub
+//
+// For the purpose of this example we will create temporary digital twin using random Ids.
+// We have to make sure these model Ids are unique within the DT instance so we use generated UUIDs.
+async function main() {
+  const modelId = "model-" + v4();
```

Comment:
minor suggestion
```suggestion
  const modelId = `model-${v4()}`;
```

Reply:
Accepted

---
In reply to: [447257580](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r447257580) [](ancestors = 447257580)